cpp/tests/copying/segmented_gather_list_tests.cpp

```diff
@@ -44,9 +45,7 @@ template <typename T>
 using LCW = cudf::test::lists_column_wrapper<T, int32_t>;
 using cudf::lists_column_view;
 using cudf::lists::detail::segmented_gather;
-using cudf::test::iterators::no_nulls;
-using cudf::test::iterators::null_at;
-using cudf::test::iterators::nulls_at;
+using namespace cudf::test::iterators;
```

Comment:
Could you remove the `class SegmentedGatherTestList` from above? It does not appear to be used anywhere.

Reply:
Done.